docs/how-to-guides/block-tutorial/extending-the-query-loop-block.md

```diff
@@ -192,6 +192,8 @@ Notice that we have also disabled the `postType` control. When the user selects
 Because our plugin uses custom attributes that we need to query, we want to add our own controls to allow the users to select those instead of the ones we have just disabled from the core inspector controls. We can do this via a [React HOC](https://reactjs.org/docs/higher-order-components.html) hooked into a [block filter](https://developer.wordpress.org/block-editor/reference-guides/filters/block-filters/), like so:
 
 ```jsx
+const { InspectorControls } = wp.blockEditor;
```

Comment:
I think should be using the import statement here, most of the examples follow this approach.

```suggestion
import { InspectorControls } from '@wordpress/block-editor';
```